README.md

```diff
@@ -382,6 +382,28 @@ class MyConfiguration {
 }
 ```
 
+### Disable worker
+
+If you need to disable worker you can do it using properties:
```

Comment:
It would be helpful to mention why this feature is helpful (I found the comment in the issue helpful to understand the why)

Reply:
Done 👍